build/images/Dockerfile.build.windows

```diff
@@ -44,3 +47,10 @@ COPY --from=antrea-build-windows  /antrea/bin/antrea-cni.exe /k/antrea/cni/antre
 
 RUN mkdir C:\k\antrea\utils
 COPY --from=antrea-build-windows /wins/wins.exe /k/antrea/utils/wins.exe
+
+COPY --from=windows-ovs /Windows/System32/vcruntime140.dll /Windows/System32/
```

Comment:
have we checked if this file is also the only dependency to run ovsdb-server and ovs-vswitchd?

Reply:
I have verified that this file is not the only dependency, we need other two following dll files as well to run ovs commands.

Reply:
So do you mean we can run ovs-vswitchd and ovsdb-server inside nano-server as long as vcruntime140 and the two ssl dll files exist?

Reply:
yes

Reply:
@Atish-iaf ,
there are multiple vc files in system32 folder added after vc installation, can you check if other vc files were already present !

 -a----        5/10/2023   7:01 AM         414104 vcamp140.dll
-a----        5/10/2023   7:01 AM         346008 vccorlib140.dll
-a----        5/10/2023   7:01 AM         191864 vcomp140.dll
-a----        5/10/2023   7:01 AM         109440 vcruntime140.dll
-a----        5/10/2023   7:01 AM          49560 vcruntime140_1.dll

Reply:
@rajnkamr , i did not install vc, just copied 1 vc dll file and 2 ssl dll files as you can see in the code. And only these 3 dll files of vc and ssl were present in windows/system32. Others weren't.

Reply:
@Atish-iaf the question is for ovs-vswitchd, ovsdb-server, not ovs-vsctl, ovs-ofctl, could you confirm?

Reply:
Yes, `ovs-vswitchd --help` and `ovsdb-server --help` also worked and that made me think whether this image itself can also be used for antrea-ovs container ?

Reply:
I tried using antrea-windows image (having these 3 required dll files and openvswitch) for antrea-ovs container as well #5305 , it works. I think its better to use antrea-windows image for antrea-ovs container as well, it will help to avoid pulling 2 images. Or we can do this change here #5249 later.
Do you see any issues ? @tnqn @wenyingd @rajnkamr @XinShuYang

Reply:
If we can run ovsdb-server and ovs-vswitchd with container image antrea-windows, it would be best to use a single image in antrea manifest.

Reply:
@tnqn we can merge this PR first as this is ready for merge. 
To unify image we can continue on PR #5305 and try to get everything validated before release.
Thanks

Reply:
> If we can run ovsdb-server and ovs-vswitchd with container image antrea-windows, it would be best to use a single image in antrea manifest. A concern is if ovsdb-server and ovs-vswitchd can run correctly in nano-server base or not, not only with your mentioned "--help" command.

@wenyingd In PR #5305 containerd-e2e test passed, so i think it works.

Reply:
@wenyingd @tnqn Do you have more comments? Once this PR is merged, I think we can proceed with the image unification update in [#5305](https://github.com/antrea-io/antrea/pull/5305).